docs/local_development_with_docker.md

```diff
@@ -28,9 +28,10 @@ The general workflow is:
   docker-manage                      Shorthand to manage.py. inv docker.manage "[COMMAND] [ARG]"
   docker-migrate                     Updates database schema
   docker-npm                         Shorthand to npm. inv docker.npm "[COMMAND] [ARG]"
-  docker-nuke-db                     Delete your database and create a new one with fake data
+  docker-new-db                      Delete your database and create a new one with fake data
```

Comment:
```suggestion
  docker-new-db                      Delete (nuke) your database and create a new one with fake data
```

Weird workflow thing: Sometimes I search the repo for "nuke" to find this. If it's too weird, I can get over it though.

Reply:
use `inv -l` to get a list of the available commands